spec/js/ko-components/view-models/orifice-calculator-spec.coffee

```diff
@@ -1,26 +1,70 @@
 describe 'orifice-calculator-viewmodel-spec', ->
   viewModel = new OPL.KoComponents.ViewModels.OrificeCalculator()
 
+  beforeEach ->
+    viewModel = new OPL.KoComponents.ViewModels.OrificeCalculator()
+
+  itBehavesLikeMandatoryField = (field) ->
+    it 'has mandatory field validation', ->
+      field('')
+      expect(field.isValid()).toBeFalsy()
```

Comment:
isValid method returns true or false, right? so we can expect the exact value instead of a set of values that might not necessarily be correct.

expect(a).toBeFalsy() will pass if a has any of these values
- false
- 0
- ""
- undefined (note that the variable undefined isn’t always undefined!)
- null
- NaN